data-plane/receiver/src/main/java/dev/knative/eventing/kafka/broker/receiver/impl/ReceiverVerticle.java

```diff
@@ -206,6 +206,8 @@ public void stop(Promise<Void> stopPromise) throws Exception {
     @Override
     public void handle(HttpServerRequest request) {
 
+      logger.error("Received request: {}", request.path());
```

Comment:
```suggestion
```

Reply:
Or if you want this log here, could we not make is an error, maybe `logger.debug`?

Reply:
This is just for my debugging purposes. It will be removed shortly.